.github/workflows/swift_matrix.yml

```diff
@@ -149,7 +149,7 @@ jobs:
           curl -s https://raw.githubusercontent.com/apple/swift-nio/main/scripts/check-matrix-job.sh | bash
 
   windows:
-    name: Windows (${{ matrix.swift.swift_version }})
+    name: Windows (6.0)
```

Comment:
Is this just waiting to go stale?  Perhaps better not to even mention version?

Reply:
This was intentional. The problem appears that a matrix with a single element doesn't expand properly. That's why I hard coded it for now. As soon as we get 6.1 we need to expand the matrix and make this dynamic again. The idea is that the check name won't change.